OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -79,6 +79,7 @@ public interface IOrderTargeter
 		int OrderPriority { get; }
 		bool CanTarget(Actor self, Target target, List<Actor> othersAtTarget, TargetModifiers modifiers, ref string cursor);
 		bool IsQueued { get; }
+		bool OverrideSelection();
```

Comment:
Could we give this a more descriptive name at the same time?  I don't have any great ideas, but `ActionOverridesLeftClickSelection` at least makes it clear (at the expense of brevity).